pkg/crc/preflight/preflight_windows.go

```diff
@@ -70,13 +70,14 @@ var hypervPreflightChecks = []Check{
 
 		labels: labels{Os: Windows, NetworkMode: System},
 	},
-	{
-		cleanupDescription: "Removing crc's virtual machine",
-		cleanup:            removeCrcVM,
-		flags:              CleanUpOnly,
+}
 
-		labels: labels{Os: Windows},
-	},
+var cleanupCheckRemoveCrcVM = Check{
+	cleanupDescription: "Removing crc's virtual machine",
+	cleanup:            removeCrcVM,
+	flags:              CleanUpOnly,
+
+	labels: labels{Os: Windows},
```

Comment:
@anjannath we are creating `cleanupCheckRemoveCrcVM` as separate check variable to change the ordering around how it is run during cleanup?

Reply:
yes correct, to order it such, that the removeCrcVM runs before removing the machine's cache dir

Reply:
```golang
checks := []Check{}
	checks = append(checks, memoryCheck(preset))
	checks = append(checks, hypervPreflightChecks...)
	checks = append(checks, crcUsersGroupExistsCheck)
	checks = append(checks, userPartOfCrcUsersAndHypervAdminsGroupCheck)
	checks = append(checks, vsockChecks...)
	checks = append(checks, bundleCheck(bundlePath, preset))
	checks = append(checks, genericCleanupChecks...)
	checks = append(checks, daemonTaskChecks...)
	checks = append(checks, adminHelperServiceCheks...)
```

> yes correct, to order in such, that the removeCrcVM runs before removing the machine's dir

@anjannath so aren't we already remove VM before removing machine dir because removeVM is part of `hypervPreflightChecks` and dir remove is part of `genericCleanupChecks` ?

Reply:
In that case the detail why we are creating a `cleanupCheckRemoveCrcVM` as separate check in the commit log to provide a sense why we did it.

Reply:
update the commit msg, added

>  this separates out the check deleting the VM from the list of hyperv
    checks, and adds it to the list of all checks after the generic cleanup
    checks to make it run before deleting the machine's dir